Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -900,9 +900,14 @@ static String mutationObserverNotificationString()
     }
 }
 
+bool UnifiedPDFPlugin::hasSelection() const
+{
+    return m_currentSelection && ![m_currentSelection isEmpty];
+}
```

Comment:
I feel like I've seen this snippet in multiple places, but it's good that you addressed at least one dupe